src/main.js

```diff
@@ -722,15 +773,13 @@ function addEleContent(e) {
   else {
       console.warn("Node/Link NOT FOUND");
   }
-
-
 }
 
 function showContent(ele) {
-    var addressToFrame = "http://webstrates.ucsd.edu/" + ele;
-    var wrapper = document.createElement('div');
+    let addressToFrame = "http://webstrates.ucsd.edu/" + ele;
```

Comment:
Create a new constant in constant.js to represent WEBSTRATES_URL_PREFIX.

Reply:
Done